CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [0.13.0] - 2024-12-18
```

Comment:
Let's leave this under Unreleased as we will update this when we actually release!